tests/test_configuration_parser.py

```diff
@@ -35,83 +35,6 @@ class TestConfigurationParser:
-                'test': 'data/rsmtool_smEval.csv',
-                'train.lab': 'sc1',
-                'crossvalidate': 'yes',
-                'test.lab': 'r1',
-                'scale': 'scale'}
-
-        with warnings.catch_warnings():
-            warnings.filterwarnings('ignore', category=DeprecationWarning)
-
-            newdata = ConfigurationParser.normalize_config(data)
-            ok_('experiment_id' in newdata.keys())
-            assert_equal(newdata['experiment_id'], 'experiment_1')
-            assert_equal(newdata['use_scaled_predictions'], True)
-
-        # test for non-standard scaling value
-        data = {'expID': 'experiment_1',
-                'train': 'data/rsmtool_smTrain.csv',
-                'LRmodel': 'LinearRegression',
-                'scale': 'Yes'}
-        with warnings.catch_warnings():
-
-            warnings.filterwarnings('ignore', category=DeprecationWarning)
-            assert_raises(ValueError, ConfigurationParser.normalize_config, data)
-
-        # test when no scaling is specified
-        data = {'expID': 'experiment_1',
-                'train': 'data/rsmtool_smTrain.csv',
-                'LRmodel': 'LinearRegression',
-                'feature': 'feature/feature_list.json',
-                'description': 'A sample model with 9 features '
-                               'trained using average score and tested using r1.',
-                'test': 'data/rsmtool_smEval.csv',
-                'train.lab': 'sc1',
-                'crossvalidate': 'yes',
-                'test.lab': 'r1'}
-
-        with warnings.catch_warnings():
-            warnings.filterwarnings('ignore', category=DeprecationWarning)
-
-            # Add data to `ConfigurationParser` object
-            newdata = ConfigurationParser.normalize_config(data)
-            ok_('use_scaled_predictions' not in newdata.keys())
-
-    def test_load_config_from_dict(self):
-        data = {'expID': '001',
-                'train_file': 'path/to/train.tsv',
-                'test_file': 'path/to/test.tsv',
-                "flag_column": "abc",
-                "model": 'LinearRegression'}
-
-        configdir = Path('/path/to/configdir').resolve()
-        configuration = Configuration(data, configdir=configdir)
-        eq_(configuration._configdir, configdir)
-
-    def test_load_config_from_dict_no_configdir(self):
-        data = {'expID': 'test',
-                'train_file': 'path/to/train.tsv',
-                'test_file': 'path/to/test.tsv',
-                "flag_column": "abc",
-                "model": 'LinearRegression'}
-
-        configdir = Path(getcwd())
-        configuration = Configuration(data)
-        eq_(configuration._configdir, configdir)
-
-    @raises(TypeError)
-    def test_load_config_from_dict_wrong_type(self):
-        data = [('expID', 'test')]
-        configdir = 'some/config/dir'
-        Configuration(data, configdir)
```

Comment:
Hmm, something I missed the last time around, why are we removing the test functions other than `test_normalize_config()`? Don't they still apply?

Reply:
I added back the test for TypeError. 

The other two are leftovers from earlier tests and implicitly test normalization (all field names are from the old R-style config file). We are already testing for the same functionality in `test_init_default_values` (tests for initiating from dictionary with no configdir), `test_init_with_configdir_only_as_kword_argument` (tests for initiating with configdir).

Reply:
Ah, great. Makes sense. Thanks!